ts/components/core/typography/H4.tsx

```diff
@@ -13,7 +13,7 @@ type AllowedBoldColors = Extract<
 // these colors are allowed only when the weight is SemiBold
 type AllowedSemiBoldColors = Extract<
   IOColorType,
-  "white" | "bluegreyDark" | "blue"
+  "white" | "bluegreyDark" | "blue" | "bluegreyLight"
```

Comment:
Shall we summon @thisisjp to get clearance for this change in the typography system?

Reply:
In this case should be fine since we have already planned with @thisisjp the possibility to choose any `IOColors` for the typhograpy elements in https://pagopa.atlassian.net/browse/IAI-12